src/index.js

```diff
@@ -25,31 +25,47 @@ var cronofy = function (config) {
 cronofy.prototype._httpCall = function (method, path, options, callback, optionsToOmit) {
   var settings = {
     method: method,
-    path: path,
+    url: path,
+    gzip: true,
+    json: true,
     headers: {
       Authorization: 'Bearer ' + (options.access_token || options.bearer_token),
-      'Content-Type': 'application/json'
+      'Content-Type': 'application/json',
+      'User-Agent': 'Cronofy Node'
```

Comment:
For your logging/debug's sake I would advice to add the version of the module to this ;)